doc/networking.md

```diff
@@ -107,22 +108,56 @@ The composition of the body (and certain headers) varies based on the content-ty
 Processing a `/segment` request consists of the following steps:
 
 1. Verify the credentials generated by the orchestrator
-2. Verify that the job hasn't expired.
-2. Verify the segment signature from the broadcaster.
-3. Download the body
-4. Verify the keccak256 hash of the body matches `SegData.hash`
-5. Transcode the segment
-6. Return 200 if transcoding success, non-200 otherwise.
+2. Verify that the job hasn't expired and is still claimable.
```

Comment:
Since we also specify that a job must still be claimable in the workflow for `GetTranscoder`, should we define what a job being claimable means once and just reference the definition from both workflows?

Reply:
I was thinking of just pasting the definition of "claimable" at the top of the Broadcaster to Orchestrator spec for reference from both workflows descriptions

Reply:
:+1: Done